.github/workflows/android_build.yml

```diff
@@ -115,6 +123,22 @@ jobs:
         with:
           ref: ${{ inputs.branch }}
 
+      - name: 📂 Set up DEV Google Services
+        if: ${{ !startsWith(inputs.gradlew-command, 'false') }}
+        uses: davidSchuppa/base64Secret-toFile-action@v3
+        with:
+            secret: ${{ env.CI_DEVELOP_GOOGLE_SERVICES_FILE }}
+            filename: google-services.json
+            destination-path: ./app
+      
+      - name: 📂 Set up PROD Google Services
+        if: ${{ !startsWith(inputs.gradlew-command, 'false') }}
+        uses: davidSchuppa/base64Secret-toFile-action@v3
+        with:
+            secret: ${{ env.CI_PRODUCTION_GOOGLE_SERVICES_FILE }}
+            filename: google-services.json
+            destination-path: ./app/src/release/
```

Comment:
@valentunn The dev google-services I put to common folder (`app/`) and for release build it should be overridden by `app/src/release/`